client/src/components/MultiJurisdictionAudit/RoundManagement/index.tsx

```diff
@@ -20,13 +20,15 @@ import BatchRoundDataEntry from './BatchRoundDataEntry'
 import { useAuthDataContext } from '../../UserContext'
 import useBallots, { IBallot } from './useBallots'
 import { IRound } from '../useRoundsAuditAdmin'
+import OfflineBatchRoundDataEntry from './OfflineBatchRoundDataEntry'
 
 const PaddedWrapper = styled(Wrapper)`
   flex-direction: column;
   align-items: flex-start;
-  width: 510px;
   padding: 30px 0;
 `
+// TODO
+// width: 510px;
```

Comment:
@jonahkagan Confused about what's going on here.

Reply:
the new form needs the full width. this width constraint was here to make the other views rendered in this wrapper more narrow so they would look better. we'll need to come back and style those views specifically at some point